app/components/elements/forms/select_field_component.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= tag.div class: 'mb-4', data: do %>
   <%= render Elements::Forms::LabelComponent.new(form:, field_name:, label_text: label, hidden_label:) %>
   <%= render Elements::Forms::HelpTextComponent.new(id: help_text_id, help_text:) %>
-  <%= form.select field_name, options, { prompt: }, { class: 'form-select', aria: field_aria } %>
+  <%= form.select field_name, options, { prompt: }, { class: 'form-control form-select', aria: field_aria, style: styles } %>
```

Comment:
Using both form-control and form-select doesn't seem like correct BS.

Reply:
Same as above, fixed.